### PR TITLE
Added support for InvocationTransaction/GetScript

### PR DESCRIPTION
**Pull request by Taconut**
**What current issue(s) from Github does this address?**
Fixes CityOfZion/neo-python#324

**What problem does this PR solve?**
neo-project/neo has an implementation of InvocationTransaction.GetScript, but neither this nor neo-python contain it.

**How did you solve this problem?**
I created a new InvocationTransaction file and added GetScript.

**How did you make sure your solution works?**
I verified that the InvocationTransaction.py file conforms to the format of the other files in /interop/Neo/

### CODE REVIEW

File: boa/interop/Neo/InvocationTransaction.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+
+class InvocationTransaction:
+
+    @property
+    def GetScript(self):
```
Comment (localhuman) [inline boa/interop/Neo/InvocationTransaction.py]: I would change this to the following so people can use it like so: `myscript = tx.Script`

```
@property
def Script(self)
```

**Comment by localhuman** [comment]
Thanks for the contribution! 🎉 

To get the travis build to pass, you will need to merge the most recent changes in development into this branch.

Also, make sure to run `pycodestyle boa` and fix any stlye issues that you find.

**Comment by Taconut** [comment]
Thay should take care of everything. I renamed the getter to "Script", removed the blank line at the end of the file, and pulled the recent changed from development into my branch.

**Comment by localhuman** [comment]
⭐️

**Pull request merged**